p2p/src/simulated/network.rs

```diff
@@ -106,54 +104,58 @@ impl<E: Spawner + Rng + Clock> Network<E> {
         )
     }
 
+    fn get_next_socket(&mut self) -> SocketAddr {
+        let result = self.next_addr;
+        self.next_addr.set_port(self.next_addr.port() + 1);
```

Comment:
Should we handle rollover?

Reply:
(I think it will panic by default, so if we don't want that we should handle somehow)